src/components/RocketContainer.js

```diff
@@ -3,20 +3,25 @@ import { useDispatch, useSelector } from 'react-redux';
 import { getRockets, allRockets } from '../redux/rockets/rocketsSlice';
 import RocketList from './RocketList';
 
-function RocketContainer() {
+const RocketContainer = () => {
   const rockets = useSelector(allRockets);
   const dispatch = useDispatch();
 
-  console.log(rockets);
-
   useEffect(() => {
     dispatch(getRockets());
   }, [dispatch]);
   return (
-    <>
-      <RocketList />
-    </>
+    <section className="flex flex-col gap-6 px-14 py-8">
+      { rockets.map((rocket) => (
+        <RocketList
+          key={rocket.rocketId}
+          rocketImage={rocket.rocketImage}
+          rocketName={rocket.rocketName}
+          rocketDescription={rocket.rocketDescription}
+        />
```

Comment:
Good work passing in the `props` but I think it's best if you pass in the rockets as a `prop` to RocketList.
Then in `RocketList` `map()` into rockets and render each `Rocket` component. It's kinda like the Todo React App.

So the structure looks like this:
😄 
RocketsContainer=> RocketList => Rocket

Reply:
OK! This makes sense and will make a good component logic flow! Thanks!